### DISCUSSION TITLE
Pairing a Hue Dimmer with a group of Hue Bulbs

### DISCUSSION THREAD

**Discussion by GerdRuetten** [Koenkk/zigbee2mqtt#20887] [Devices]
Dear all,

i need your help :-)

I have a Hue Dimmer and five Hue Bulbs.
Dimmer and Bulbs are integrated in z2m without problems.

I put in z2m the five bulbs in a group.

How can i now pairing the dimmer with this group over the z2m-GUI?

In the dimmer-device in the Bind-Tab i see this:

<img width="1418" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/97361636/098cb628-0cf5-4598-84d1-e374b0970685">

The docu under https://www.zigbee2mqtt.io/devices/324131092621.html#binding says, i musst unbind it from the coordinator and bind it to the lamp ... but i have a group.

Can sombody explain me, how to do it over the UI.

TIA

Best regards
Gerd

**Comment by Yoldark34** [comment]
Little late but i did it today.
https://www.zigbee2mqtt.io/devices/929002398602.html

You do not need to add the dimmer to the group. You need put the log level to debug in Settings, Settings, Advanced tab, put in debug, reboot the app.
Bind only the genScenes to the coordinator and pressed the hue button you should have identified the groupId used by the dimmer and created your group with the same groupId as in the debug logs.
If the remote is unresponsive, remove the battery, wait a little, add it back and try binding quickly.

There is no way to add the dimmer to the group in the group settings, the dimmer will call the scene action using the groupId you discovered.
You bind the dimmer to the group in the bind tab of the dimmer instead of the coordinator.
You need to go in the dimmer device on Zigbee2MQTT webPortal, in Bind tab. you remove all the bindings to the coordinator then you add the binding genLevelCtrl, genOnOff, genScenes to the group you created with the GroupId you identified, for intance "Hue Group".

Best of luck
